index.ts

```diff
@@ -0,0 +1,6 @@
+class Main {
+    public static main(): void {
+    }
+}
+
+Main.main();
```

Comment:
In the Node.js we have to use not only the entry point, but in case of CLI app, we also have to define that the main module can only be called directly `require.main === module`.

For ES6 this code looks a little different.